kogitoq/gadaq/src/main/java/life/genny/gadaq/route/Events.java

```diff
@@ -181,11 +183,17 @@ public void route(QEventMessage msg) {
 			// TODO: determine if DEF depends on DEF_USER
 			if (Prefix.PER_.equals(prefix)) {
 				kogitoUtils.triggerWorkflow(SELF, "userLifecycle", json);
+                return;
```

Comment:
We must not return after triggering userLifecycle. The message must be forwarded on to the product.